Seneca-CDOT/telescope
# Make sign up work with blogs and channels

Update the signup frontend to work with blog and channel feeds.
- Extract RSSFeeds component
- Display Feeds type: 'Blog / Youtube / Twitch'
- Conditionally validate Channels Feeds

I've tested this locally, if we can merge this soon, let's do another release to make Signup work in production. 
## Steps to test the PR

1. docker-compose up -d sso feed-discovery auth kong rest meta studio traefik 
2. pnpm dev
3. Go to localhost:8000/signup
4. Finish the sign up worflow
5. Inspect the studio http://localhost:8910/ to make sure every works
